website/api/views.py

```
@@ -929,7 +929,7 @@ class SearchHistoryApiView(APIView):
 
     def get(self, request, *args, **kwargs):
```

Ruff (0.14.7)

Unused method argument: `args`

(ARG002)